internal/pkg/template/workload.go

```diff
@@ -43,9 +43,8 @@ const (
 
 // Constants for ARN options.
 const (
-	snsArnPattern     = "arn:%s:sns:%s:%s:%s-%s-%s-%s"
-	AWSPartition      = "aws"
-	AWSChinaPartition = "aws-cn"
+	snsArnPattern = "arn:%s:sns:%s:%s:%s-%s-%s-%s"
```

Comment:
Since "arn" is an acronym and unique name we should use either all cap or all lower-case, and all cap in this case.

We haven't been perfectly consistent on this in history, but I think we've started to adapt to all-lower/all-cap at some point. 
```suggestion
	snsARNPattern = "arn:%s:sns:%s:%s:%s-%s-%s-%s"
```